firestore/integration_test.go

```diff
@@ -151,25 +154,42 @@ func createIndexes(ctx context.Context, dbPath string) {
 				Fields:     adminPbIndexFields,
 			},
 		}
-		go func(req *adminpb.CreateIndexRequest) {
-			op, err := iAdminClient.CreateIndex(ctx, req)
-			if err != nil {
-				log.Fatalf("CreateIndex: %v", err)
+		go func(req *adminpb.CreateIndexRequest, i int) {
+			op, createErr := iAdminClient.CreateIndex(ctx, req)
+			if createErr != nil {
+				log.Fatalf("CreateIndex: %v", createErr)
 			}
 
-			_, err = op.Wait(ctx)
-			if err != nil {
-				log.Fatalf("Wait: %v", err)
+			createdIndex, waitErr := op.Wait(ctx)
+			if waitErr != nil {
+				log.Fatalf("Wait: %v", waitErr)
+			} else {
+				indexNames[i] = createdIndex.Name
 			}
 			createIndexWg.Done()
-		}(req)
+		}(req, i)
 	}
 	createIndexWg.Wait()
 }
 
+// deleteIndexes deletes composite indexes created in createIndexes function
+func deleteIndexes() {
+	ctx := context.Background()
+
+	for _, indexName := range indexNames {
+		err := iAdminClient.DeleteIndex(ctx, &adminpb.DeleteIndexRequest{
+			Name: indexName,
+		})
+		if err != nil {
+			log.Printf("Failed to delete index \"%s\": %+v\n", indexName, err)
+		}
+	}
+}
+
 func cleanupIntegrationTest() {
 	if iClient != nil {
 		// TODO(jba): delete everything in integrationColl.
```

Comment:
Does this TODO have any significance? If not, I'd suggest deleting it since jba no longer works on this project.

Reply:
Removed todo. Added cleanup of collection